pkgs/by-name/na/nautilus/wrapper.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  makeWrapper,
+  symlinkJoin,
+  nautilus,
+  nautilusExtensions,
+}:
+
+symlinkJoin {
+  name = "nautilus-with-extensions-${nautilus.version}";
+
+  paths = [ nautilus ] ++ nautilusExtensions;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  postBuild = ''
+    wrapProgram "$out/bin/nautilus" \
+      --set "NAUTILUS_4_EXTENSION_DIR" "$out/lib/nautilusx-3"
```

Comment:
This seems a bit like a version mismatch..

Reply:
Good catch

```suggestion
      --set "NAUTILUS_4_EXTENSION_DIR" "$out/lib/nautilus/extensions-4"
```